examples/docker-dektop/README.md

```diff
@@ -0,0 +1,32 @@
+# Docker Desktop (Windows) Example
+
+This example is tested on docker desktop v4.26.1.
+
+## Install [Docker Desktop](https://docs.docker.com/desktop/install/windows-install/)
+
+Once docker desktop is installed, enable kubernetes extension in the settings.Ensure both docker engine & kubernetes are green before proceeding.
+![Kubernetes extension](image.png)
+
+## Install kubectl & helm
+
+## Install NGINX ingress controller
+
+```
+helm upgrade --install ingress-nginx ingress-nginx \
+  --repo https://kubernetes.github.io/ingress-nginx \
+  --namespace ingress-nginx --create-namespace
+```
+
+## Navigate to docker-desktop/datadir and run helm install
+
+```
+ helm upgrade --install --values=.\values.yaml ogsdocker .
```

Comment:
@danduk82 just noticed, there typo with the release name it should be "ogsv1"
helm upgrade --install --values=.\values.yaml ogsv1 .